lib/javascript/JavascriptModulesPlugin.js

```diff
@@ -1089,9 +1089,13 @@ class JavascriptModulesPlugin {
 					}
 				}
 				if (runtimeRequirements.has(RuntimeGlobals.onChunksLoaded)) {
-					buf2.push(
-						`__webpack_exports__ = ${RuntimeGlobals.onChunksLoaded}(__webpack_exports__);`
-					);
+					if (runtimeRequirements.has(RuntimeGlobals.prefetchChunk)) {
+						buf2.push(`${RuntimeGlobals.onChunksLoaded}();`);
+					} else {
+						buf2.push(
+							`__webpack_exports__ = ${RuntimeGlobals.onChunksLoaded}(__webpack_exports__);`
+						);
+					}
```

Comment:
I thought `onChunksLoaded` fired by webpackPrefetch should not overwrite `__webpack_exports__` because `prefetchChunkHandlers` return void, so I fixed issue 12993 by changing how to call `onChunksLoaded` with prefetchChunk.

If my approach is not correct or a test case is not enough, please free to close this PR or request changes.